File: code-coaches/easy/argentina/Argentina.cs
using System;

public class Argentina
{
    static string LowerCurrencyAfterConversion(decimal ars, decimal usd)
    {
        decimal arsToUsd = 0.02m;
        decimal convertedUsd = ars * arsToUsd;

        if (convertedUsd > usd)
            return "Dollars";
        else
            return "Pesos";
    }

    static void Main(string[] args)
    {
        decimal ars, usd;

        ars = decimal.Parse(Console.ReadLine());
        usd = decimal.Parse(Console.ReadLine());

        Console.WriteLine(LowerCurrencyAfterConversion(ars, usd));
    }
}

File: code-coaches/easy/argentina/argentina.c
#include <stdio.h>
#include <string.h>

char* lower_currency_after_conversion(double ars, double usd)
{
    double arsToUsd = 0.02; // exchange rate
	double convertedUsd = ars * arsToUsd;

	if (convertedUsd > usd)
		return "Dollars";
	else
		return "Pesos";
}

int main()
{
	double ars; // Argentine peso
	scanf("%lf", &ars);

	double usd; // US dollar
	scanf("%lf", &usd);

    printf("%s", lower_currency_after_conversion(ars, usd));

	return 0;
}